packages/tracing/src/span.ts

```diff
@@ -367,15 +374,46 @@ export class Span implements SpanInterface {
    *
    * @returns modified and immutable baggage object
    */
-  private _getBaggageWithSentryValues(baggage: Baggage = createBaggage({})): Baggage {
+  private _populateBaggageWithSentryValues(baggage: Baggage = createBaggage({})): Baggage {
+    // Because of a cicular dependency, we cannot import the Transaction class here, hence the type casts
```

Comment:
As discussed: Root cause of the circular dependency is, that we some of the Dynamic Sampling Context functionality on the `Span` class, while it should probably all live on the `Transaction` class.

We will clean this up in a follow-up PR so no required action on this for now.

Reply:
Almost done with it already. Will make a PR soon